OracleWebLogic/samples/12213-coherence-domain-in-image-wdt/README.md

```diff
@@ -0,0 +1,103 @@
+Sample Image with a WLS Domain and Coherence
+===============================
+
+This sample demonstrates how to create a WebLogic 12.2.1.3 domain with a Coherence cluster using 
+WebLogic Deploy Tooling (WDT).  The sample also includes a Coherence proxy server that runs 
+inside the domain, along with a proxy client to access a Coherence cache.  The domain is run 
+in docker containers and the Coherence cache is accessed using a proxy client running outside of docker.  
+The image created in this sample can also be used to create a domain using Kubernetes.  
+For more information on WDT, refer to https://github.com/oracle/weblogic-deploy-tooling.  For more 
+information on using WebLogic on Kubernetes, see https://github.com/oracle/weblogic-kubernetes-operator.
+
+### WDT Model File and Property Files
+
+The image created in this sample, using WDT, has both the WebLogic binary home and domain home.  The WDT input file, `cohDomain.yaml`, 
+describes the configuration of the domain, which specifies a WebLogic dynamic cluster with a maximum of
+five managed servers.  The domain will also have a Coherence cluster scoped to the servers running in that dynamic cluster.
+
+The sample includes `properties` files needed to create the domain.  The files already have
+the correct values for the sample, you do not need to modify them. 
+The files are `adminuser.properties` and the `adminpass.properties` in the `properties/docker_build` directory.
+Care should be taken to secure the credentials that are present in the model. The ADMIN credential 
+attributes in the sample model have a file token referencing a special property file. Each special 
+property file must only contain a single property and can be created and modified using a text editor. 
+
+The sample also provides `security.properties` in the `properties/docker-run` directory. This file contains 
+the admin credentials and additional properties used to customize the WebLogic Server start.
+The ADMIN credentials are necessary to start the Administration or Managed Server in a Docker container.   
+It is the responsibility of the user to manage this volume, and the security.properties, in the container.
+
+**Note**: Oracle recommends that the `adminpass.properties`, `adminuser.properties`, and `security.properties` files 
+be deleted or secured after the image is built and the WebLogic Server is started so that the user name 
+and password are not inadvertently exposed.
+
+## How to Build and Run
+
+**NOTE:** The image is based on the following WebLogic Server image: 
+    `container-registry.oracle.com/middleware/weblogic:12.2.1.3`  
```

Comment:
build image under OracleWebLogic/dockerfiles/12.2.1.3 or pull image from OCR and run docker target

Reply:
fixed